lib/DxcSupport/FileIOHelper.cpp

```diff
@@ -18,6 +18,7 @@
 #include "dxc/dxcapi.h"
 
 #include <algorithm>
+#include <assert.h>
```

Comment:
The issue here is that our header includes aren't clean. Global.h actually introduces the dependency on the assert macro, but including an assert header there breaks things because of conflicting definitions of `assert` across the codebase.

If you're okay with it I'd like to do a cleanup of `assert` macros after I get the clang-format in.